tests/test_cmake_build/subdirectory_target/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ endif()
 
 project(test_subdirectory_target CXX)
 
-add_subdirectory(${PYBIND11_PROJECT_DIR} pybind11)
+add_subdirectory("${pybind11_SOURCE_DIR}" pybind11)
```

Comment:
How are these two variables different?

Reply:
This is now the same name as the original variable, rather than being renamed, so it works when added from the main file as well.